shell/browser/api/electron_api_base_window.cc

```diff
@@ -1055,9 +1057,10 @@ void BaseWindow::ResetBrowserViews() {
                            v8::Local<v8::Value>::New(isolate(), item.second),
                            &browser_view) &&
         !browser_view.IsEmpty()) {
-      window_->RemoveBrowserView(browser_view->view());
-      if (browser_view->web_contents())
+      if (browser_view->web_contents()) {
         browser_view->web_contents()->SetOwnerWindow(nullptr);
+        window_->RemoveBrowserView(browser_view->view());
```

Comment:
shouldn't this be called before setting the owner window to `nullptr` ?

Reply:
ah dangit you're right